FactoryDesignPattern.java

```diff
@@ -0,0 +1,97 @@
+import java.util.Scanner;
+
+public class FactoryDesignPattern {
+    public static void main(String[] args) {
+        Logistics l= new Logistics();
+        try {
+            l.dealWithLogistic();
+        } catch (Exception e) {
+            System.out.println(e.getMessage());
+        }
+
+    }
+}
+
+class Logistics {
+    void dealWithLogistic() throws Exception{
+        Scanner sc= new Scanner(System.in);
+        String transportMode= sc.nextLine();
+
+        TransportLogistic t;
+        switch(transportMode){
+            case "ship":{
+                t= new ShipLogisticc();
+            }
+            break;
+            case "truck":{
+                t= new TruckLogistic();
+            }
+            break;
+            default:{
+                throw new Exception("go deliver it on your back!!!!");
+            }
+        }
+        System.out.println(t.findDeliveryPrice());
+        t.showDeliveryMessage();
+
+    }
+}
+
+//creator class
```

Comment:
create a readme, add a simple description for the following
- creator
- product
- concreate creator etc.

Reply:
done